src/pybind/mgr/nfs/ganesha_conf.py

```diff
@@ -295,12 +301,13 @@ def to_fsal_block(self) -> RawBlock:
     @classmethod
     def from_dict(cls, fsal_dict: Dict[str, str]) -> 'RGWFSAL':
         return cls(fsal_dict['name'],
+                   fsal_dict['cmount_path'],
                    fsal_dict.get('user_id'),
                    fsal_dict.get('access_key_id'),
                    fsal_dict.get('secret_access_key'))
 
     def to_dict(self) -> Dict[str, str]:
-        r = {'name': self.name}
+        r = {'name': self.name, 'cmount_path': self.cmount_path}
```

Comment:
```suggestion
        r = {'name': self.name}
        if self.cmount_path:
            r['cmount_path'] = self.cmount_path
```

Reply:
Okay! But anyways default value we're setting as `/` so it'll never be `None` right?

Reply:
If we read and parsed an old object from a previous build/version of ceph it may not have a cmount_path. I'd like to not break upstream users that might have deployed nfs in reef for example, unless absolutely necessary.

Reply:
Ah! I see the point here, completely missed! Thanks for clarification @phlogistonjohn. I'll update the PR